docs/tutorial/1-serialization.md

```diff
@@ -77,7 +77,7 @@ For the purposes of this tutorial we're going to start by creating a simple `Sni
 We'll also need to create an initial migration for our snippet model, and sync the database for the first time.
 
     python manage.py makemigrations snippets
-    python manage.py migrate
+    python manage.py migrate snippets
```

Comment:
Do we need `manage.py migrate snippets` here, or can it just be `manage.py migrate`?

Reply:
just manage.py migrate works, but i added the snippets to follow the form 
above `makemigrations snippets` (just manage.py makemigrations is also fine)